src/main/java/frc/robot/constants/RobotMap.java

```diff
@@ -6,30 +6,24 @@
  * the wiring easier and significantly reduces the number of magic numbers
  * floating around.
  */
-public abstract class RobotMap {
-    public CAN can = new CAN();
-    public PCM pcm = new PCM();
-    public DIO dio = new DIO();
-    public PWM pwm = new PWM();
-
+public class RobotMap{
     // TODO: Set variables for hardware components
 
-    public class CAN {
+    public static class CAN{
 
     }
 
-    public class PCM {
+    public static class PCM{
 
     }
 
-    public class DIO {
+    public static class DIO{
 
     }
 
-    public class PWM {
-        public LED led = new LED();
-        public class LED{
-            public int LED_CONTROLLER;
+    public static class PWM{
+        public static class LED{
+            public static final int LED_CONTROLLER = -1;
```

Comment:
A. the port should **not** be static because if the port itself is static that
pretty sure it isn't -1

Reply:
it's -1 because it's not set yet...